common.mk

```diff
@@ -484,6 +500,7 @@ endif
 endif
 endif
 endif
+endif
```

Comment:
Maybe `endif #tmux` to help not getting to lost in the condition cases ?
```make
...
check-terminal := @echo "Error: could not find gnome-terminal, konsole nor xterm" ; false
endif
endif
endif
endif #tmux
endif #LAUNCH_TERMINAL

Reply:
Good idea 👍